### ISSUE TITLE
Chatterino is offset in the native browser extension

### ISSUE THREAD

**Issue by ichiuto** [Chatterino/chatterino-browser-ext#89]
I tried resetting the app, uninstalled then reinstalled the browser extension. Still offset.
![2023-03-27](https://user-images.githubusercontent.com/129070134/227954341-2d0c8406-5fb9-44ea-9964-fbfd8b4b36a8.png)
![2023-03-27 (1)](https://user-images.githubusercontent.com/129070134/227954691-cec1bc77-c28e-458b-8dca-2c29bfdc0b82.png)

**Comment by Felanbird** [comment]
You don't have to open a new issue everytime you want to change something, you can just edit the old one

**Comment by Nerixyz** [comment]
Let's track this in here.

To make reproduction easier:

* What browser are you on (looks like Chrome)?
* What Chatterino version are you running?
* Which extensions are you running (on Twitch)?
* Are you using any other browser features, such as a bookmarks bar?

I can't reproduce this with the latest Chatterino version and neither Firefox Nightly nor Google Chrome.

**Comment by ichiuto** [comment]
@Felanbird Sorry about that. It gave me an error the first few times I submitted and I couldn't find my post, so I thought it didn't go through. Sorry if it was accidentally spammed!

@Nerixyz 
- Yes, it is Chrome for Windows 11
- I am running the most recent version 2.4.2 
- I am running AdBlock, Volume Master, and BetterTTV extensions
- I am using the bookmarks bar

It only started this early this morning. I tried restarting everything as well to no avail.

**Comment by Mudias** [comment]
The same problem started happening to me as well this morning, I tried reinstalling the extension, using different browsers and different chatterino versions but the problem still happens no matter what. The current Chatterino version I'm running is 7.4.2

**Comment by Oblivious322** [comment]
Same problem

Using chrome on windows 10
Tried Nightly release, normal 2.4.2 and 2.4.1
running adblock, bttv, 7tv, ffz
I use bookmarks bar

**Comment by Nerixyz** [comment]
I reproduced it. It seems like this was introduced in #28 and only shows up if you're using a high-dpi monitor. I'll try to fix it.

**Comment by Nerixyz** [comment]
As you can see, I made a fix. Until this lands in the add-on stores, you can manually load the extension. _This isn't persistent across browser restarts._

**Chrome:** You need to have a copy/clone of [my branch](https://github.com/Nerixyz/chatterino-browser-ext/tree/fix/high-dpi) locally (or the master branch once the PR is merged) e.g. by clicking on `Code > Download ZIP`. Go to `chrome://extensions` and enable developer mode. Then click on `Load Unpacked` and select the `src` folder from your local copy.

**Firefox:** You can either use a copy like on Chrome, or use a zip file of the `src` folder (current: [chatterino-browser-ext-fix.zip](https://github.com/Chatterino/chatterino-browser-ext/files/11080674/chatterino-browser-ext-fix.zip)). Go to `about:debugging` and click on `This Firefox`. There, click on `Load Temporary Add-On...`. In the dialog, either select the zip file or select `src/manifest.json` from your copy.

**Comment by Nerixyz** [comment]
> I'm not sure if the issue is just for me still, but when I followed the instructions to update manually it still had an offset?

Are you sure you disabled the regular extension? You can only run one extension at a time.

**Comment by ichiuto** [comment]
OK, I disabled the regular extension. Now, this shows? The app won't load?

![2023-03-27 (6)](https://user-images.githubusercontent.com/129070134/228024901-aec7c90a-71dd-45f0-a2f4-6ee6936e36e7.png)

**Comment by megamen128** [comment]
I loaded extension from bugfix branch and have the same issue, chatterino is not detected on 4k nor 1080p monitor. Official version works well on 1080p monitor and has this offset on 4k monitor

**Comment by megamen128** [comment]
I added following property to manifest.json and it works now :)
`"key": "MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAjtsADaNVdZWOwRl7Sdth0tv2vq5H9t9jVaBK8FQbs6fmZpAfgt5U4LBRI6RiI6TocxOYY0KNEdNUNmC58+3UoDn94jDViaOVszMeJeIIozDtfcpje3I5DC6pgNGJdRVwK/JMbiL+jyfc3a2ztZE36ufhfpVke192vuw12TmhYyVq+5P7c5/E2Dc8CFwDZBxLwXXlnyTZIWEHGadaWoqY5Jnjjmif5I6O4WG35Kf+GAIKNqiq03nxicpXm1UiujHxG1R5YtD0CfqjutaxsQGt7H2qWksrTC7x5EqoUI1IgPKsMYXSc/yWmkdm/viEdC0LeyYM+ivaOJRn5fECWNMrCwIDAQAB",
` 

[source](https://stackoverflow.com/a/21500707)
